src/popup/router/components/Modals/Help.vue

```diff
@@ -0,0 +1,47 @@
+<template>
```

Comment:
I don't think that this version is good solution either, we have 2 or even 3 more places with hints.
Will you propose to v-else the same code 3 additional times?

Reply:
Well, the reason for this, as well as the solution before is to avoid using dynamic localization keys.

It is caused by the need to have new lines in the modal message which is achieved by component interpolation: https://kazupon.github.io/vue-i18n/guide/interpolation.html.

I'm open for better solutions, of course.

Reply:
Something similar to this https://github.com/aeternity/aepp-base/blob/develop/src/components/Guide.vue,
but simplier